src/Components/ColorForm/ColorForm.jsx

```diff
@@ -38,7 +39,11 @@ export default function ColorForm({
         <ColorInput id="contrastText" defaultValue={initialData.contrastText} />
       </label>
       <br />
-      <button>ADD COLOR</button>
+      {isEditing ? ( // Update Color Button nur im Bearbeitungsmodus
+        <button type="submit">Update Color</button>
+      ) : (
+        <button type="submit">Add Color</button> // Add Color Button im normalen Modus
+      )}
```

Comment:
You can shorten this: 
`<button>{ isEditing ? "Update Color" : "Add Color" } </button>`

You can also omit the `type="submit"` because this is the default type.